primitive-types/tests/conversion.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Parity Technologies
```

Comment:
This file should probably be named `fp_conversions.rs` or something that conveys the content better.